target/linux/ath79/dts/ar9344_meraki_z1.dts

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+#include "ar9344.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "meraki,z1", "qca,ar9344";
+	model = "Cisco Meraki Z1";
+
+	aliases {
+		led-boot = &led_power_orange;
+		led-failsafe = &led_power_orange;
+		led-upgrade = &led_power_orange;
+		label-mac-device = &eth0;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "Reset button";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_orange: power_orange {
+			label = "orange:power";
```

Comment:
Use function/color syntax
```
			function = LED_FUNCTION_POWER;
			color = <LED_COLOR_ID_ORANGE>;
```

Reply:
Ah, forgot about that. Was lying in my tree for so long xD

Reply:
And added `panic-indicator;` for a good measure.